### ISSUE TITLE
Confusion about the usage of log_softmax function.

### ISSUE THREAD

**Issue by tengteng95** [fxia22/pointnet.pytorch#27]
Thanks for your great work! I have some doubts about the usage of **log_softmax** function.
As written in the 104th line of the file **pointnet.py**
     return F.log_softmax(x, dim=0), trans
Actually, Dim0 means the dimension of batch. However, the log_softmax should be applied along the channels.
   So I think the parameter **dim** should be set as -1. Am I right or I miss something important? Looking forward for your response!

**Comment by yhcao6** [comment]
I also think so

**Comment by fxia22** [comment]
This issue is fixed in lastest master branch, thanks for pointing it out.
